src/evict/evict_lru.c

```diff
@@ -734,6 +734,24 @@ __evict_update_work(WT_SESSION_IMPL *session)
         LF_SET(WT_EVICT_CACHE_UPDATES);
     }
 
+    /*
+     * If application threads are blocked by data in cache, track the fill ratio.
+     *
+     */
+    double cache_fill_ratio = bytes_inuse / bytes_max;
+    bool evict_is_hard = LF_ISSET(WT_EVICT_CACHE_UPDATES_HARD) ||
+      LF_ISSET(WT_EVICT_CACHE_DIRTY_HARD) || LF_ISSET(WT_EVICT_CACHE_CLEAN_HARD);
+    if (evict_is_hard) {
+        if (cache_fill_ratio < 0.25)
+            WT_STAT_CONN_INCR(session, cache_eviction_app_threads_fill_ratio_lt_25);
```

Comment:
Unrelated to this change, after a discussion with Vamsi, we feel that this stat `cache_eviction_trigger_clean_reached` not actually represent the clean threshold. It represents the cache fill/usage ratio. How about renaming the above stat to `cache_eviction_trigger_reached`?